### PR TITLE
refactor: use built-in Laravel ascii function

### PR DESCRIPTION
**Pull request by asbiin**
You fool, don't forget these steps:

- [x] Unit tests
- [ ] Tests with Cypress
- [ ] Documentation
- [ ] Dummy data

### CODE REVIEW

File: app/Http/Controllers/Company/Employee/EmployeeSkillController.php [hunk 1]
```diff
@@ -26,7 +27,7 @@ public function search(Request $request): JsonResponse
         $loggedCompany = InstanceHelper::getLoggedCompany();
 
         $name = $request->input('searchTerm');
-        $name = StringHelper::removeLettersWithAccent($name);
+        $name = Str::of($name)->ascii();
```
Comment (djaiss) [inline app/Http/Controllers/Company/Employee/EmployeeSkillController.php:30]: Are you sure of this change? Is it really a replacement?

File: tests/Unit/Helpers/StringHelperTest.php [hunk 2]
```diff
@@ -20,15 +20,4 @@ public function it_returns_the_parsed_content(): void
             StringHelper::parse($content)
         );
     }
-
-    /** @test */
-    public function it_removes_the_accents_from_a_string(): void
```
Comment (djaiss) [inline tests/Unit/Helpers/StringHelperTest.php:25]: We still need a test that makes sure that we don’t store a string with accents.

File: app/Helpers/NotificationHelper.php [hunk 3]
```diff
@@ -46,87 +46,89 @@ public static function process(Notification $notification): string
...
-
-        if ($notification->action == 'expense_rejected_by_accounting') {
-            $sentence = trans('account.notification_expense_rejected_by_accounting', [
-                'title' => $notification->object->{'title'},
-            ]);
-        }
-
-        if ($notification->action == 'employee_allowed_to_manage_expenses') {
-            $sentence = trans('account.notification_employee_allowed_to_manage_expenses', []);
+        switch($notification->action ) {
```
Comment (djaiss) [inline app/Helpers/NotificationHelper.php:49]: Man this is SO much better. I hated this list of `ifs` for months and months.

**Review by djaiss — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Company/Employee/EmployeeSkillController.php (hunk 1 above)
Comment (asbiin) [reply app/Http/Controllers/Company/Employee/EmployeeSkillController.php:30]: It has to be tested ;)

File: tests/Unit/Helpers/StringHelperTest.php (hunk 2 above)
Comment (asbiin) [reply tests/Unit/Helpers/StringHelperTest.php:25]: Sure! I though about it, then I was too lazy to do it, but we definitively need a (multiple?) test

Comment (asbiin) [reply tests/Unit/Helpers/StringHelperTest.php:25]: OK I added 1 test

**Pull request merged**